plugins/woocommerce-admin/client/layout/index.js

```diff
@@ -51,6 +51,20 @@ const WCPayUsageModal = lazy( () =>
 	)
 );
 
+const LayoutContextPrototype = {
```

Comment:
Very cool!  I really like this level of information.

My one concern with this is if we inadvertently wrap the sidebar in another context, it might lead to different context props that are not easily discovered until later.

For example, right now we have `overview/activity-panel` but if we eventually wrapped the sidebar in another layout context we could end up with `overview/sidebar/activity-panel`.

But this may not be an issue for tracks (cc @pmcpinto) and we could also avoid this with some e2e tests.

Reply:
Definitely a good thought, and one that might be worth addressing sooner rather than later. We could always just check the `path` to see what it includes, and derive a string from that, or structure things slightly differently.

That said, I've jotted this down on my own to-do to think about a bit more and I'll get this merged for now.